### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [pages-themes/minimal#49]
The user @carlsonsantana validate your site template "https://pages-themes.github.io/minimal/" and found these accessibility errors:
* Page https://pages-themes.github.io/minimal/:
  * Issue *7c9716ac-0cf1-4fbd-8e13-871ae1cfe88a*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.49:1. Recommendation: change background to #f9f9f9.
    * **Selector**: `html > body > div > section > div:nth-child(11) > div > pre > code > span:nth-child(4)`
    * **Context**: `<span class="no">Dependencies</span>`
  * Issue *9fbcf03e-5945-4d98-b526-d990e07ac2da*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.49:1. Recommendation: change background to #f9f9f9.
    * **Selector**: `html > body > div > section > div:nth-child(11) > div > pre > code > span:nth-child(2)`
    * **Context**: `<span class="no">GitHubPages</span>`
  * Issue *887f35e1-56b9-4be8-92b8-831dac729f70*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.72:1. Recommendation: change text colour to #737362.
    * **Selector**: `html > body > div > section > div:nth-child(11) > div > pre > code > span:nth-child(1)`
    * **Context**: `<span class="c1"># Ruby code with syntax highlig...</span>`
  * Issue *bd91bd1f-5ead-4f55-96f4-cf663de1b80c*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.72:1. Recommendation: change text colour to #737362.
    * **Selector**: `html > body > div > section > div:nth-child(10) > div > pre > code > span:nth-child(1)`
    * **Context**: `<span class="c1">// Javascript code with syntax ...</span>`
  * Issue *22afde61-cd46-4cbb-be9d-f65147e6a502*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.48:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > header > p:nth-child(4) > a > small`
    * **Context**: `<small>pages-themes/minimal</small>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com//validation/validations/c0756cc0-5a68-4605-8227-9e7c3cfa2124/).
